### PR TITLE
SVCS-1828 Refactored execute endpoint to functions service

### PR DESCRIPTION
**Pull request by chad-pelly**

### CODE REVIEW

File: tests/unit/tasks/execute.ts
Comment (wwaaijer-exh) [inline tests/unit/tasks/execute.ts:1]: (minor) Would it make sense to also move this to `tasks/functions/execute`. To match the file/logical structure of the source code

**Review by wwaaijer-exh — APPROVED** [review]

**Review by AaronTrippaers — APPROVED** [review]
Looks good!

File: tests/unit/tasks/functions/post.ts
Comment (wwaaijer-exh) [inline tests/unit/tasks/functions/post.ts:1]: (minor) `tasks/functions/execute` like we do in our services or `tasks/functions/execute/post`? `tasks/functions/post` would be the function creation endpoint/method

Comment (wwaaijer-exh) [reply tests/unit/tasks/functions/post.ts:1]: Totally agree! (Even for the services I might have voted for that option 😄)

**Review by wwaaijer-exh — APPROVED** [review]

**Pull request merged**